lib/metadata/mongoclient/backend.js

```diff
@@ -77,6 +84,7 @@ class MongoClientInterface {
         // initialize this backend
         MongoClient.connect(mongoUrl, (err, client) => {
             if (err) {
+                this.logger.error('error connecting to mongo server', err);
```

Comment:
For werelogs logging, the pattern is to have the second argument as an object containing additional information. For example, `...mongo server', { error: err });`. I guess I would assume we should maintain that for these changes as well?